src/library_html5_webgl.js

```diff
@@ -363,6 +363,9 @@ var LibraryHtml5WebGL = {
 #endif
 
     var ext = context.GLctx.getExtension(extString);
+    if (extString == 'WEBGL_multi_draw') {
+      context.GLctx.multiDrawWebgl = ext;
```

Comment:
why do we need this for this extension but not others? a comment could help here.

Reply:
comments added